### PR TITLE
Feature/add new logic for create

### PR DESCRIPTION
**Pull request by yeyuexia**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -29,6 +29,11 @@ signing {
     sign configurations.archives
 }
 
+ext {
```
Comment (elvis-liu) [inline build.gradle]: should be in local gradle.properties file

File: build.gradle [hunk 2]
```diff
@@ -68,6 +73,11 @@ uploadArchives {
                         name 'Elvis Liu'
```
Comment (elvis-liu) [inline build.gradle:68]: should not change this

File: src/main/java/com/exmertec/dummie/DummyBuilderFactory.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package com.exmertec.dummie;
+
+
+import com.exmertec.dummie.cache.DummyCache;
+import com.exmertec.dummie.cache.impl.DefaultCache;
+import com.exmertec.dummie.cache.impl.LevelCache;
+
+public class DummyBuilderFactory {
```
Comment (elvis-liu) [inline src/main/java/com/exmertec/dummie/DummyBuilderFactory.java:8]: As discussed, no need another builder factory, should use "convention over configuration"

File: build.gradle (hunk 2 above)
Comment (yeyuexia) [inline build.gradle:68]: 这个可以改的，试过了没问题

File: src/main/java/com/exmertec/dummie/DummyBuilderFactory.java (hunk 3 above)
Comment (yeyuexia) [inline src/main/java/com/exmertec/dummie/DummyBuilderFactory.java:8]: i think it depends on which to config.The cache that DummyBuilder used is important and unchanged so could be provided when inited

File: build.gradle (hunk 1 above)
Comment (yeyuexia) [inline build.gradle]: done

**Pull request merged**